dll/cpl/liccpa/lang/ro-RO.rc

```diff
@@ -1,33 +1,35 @@
 /*
- * FILE:       dll/cpl/liccpa/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS Exploitation System
```

Comment:
The... what? Where do you even get such naming? Maybe I overlooked it, but @GeoB99 🤨

Reply:
lmao

Reply:
Sorry, I made a mistake. What is the correct name? I can correct it.

Reply:
This is what I found on the internet.

Reply:
First rule is to search in our source code, and also whether there are other suitable instances in the module of interest (liccpa): https://git.reactos.org/?p=reactos.git;a=blob;f=dll/cpl/liccpa/liccpa.c;h=50b4fbb4266741e510e0839d0c6c342cea964b49;hb=e8cf32cf15b25cc0d653ee898768e14dd7ba0d36